pkg/interface/src/logic/state/local.tsx

```diff
@@ -40,7 +40,8 @@ const useLocalState = create<LocalState>(persist((set, get) => ({
     }
   })),
   set: fn => set(produce(fn))
-}), {
+  }), {
+  blacklist: ['suspendedFocus', 'toggleOmnibox'],
```

Comment:
`omniboxShown` is a boolean, `toggleOmnibox` is a function. Seems fine to serialise a boolean imo

Reply:
It's not as crucial as the underlying issue, but I think it would be weird to load Landscape with the omnibox showing, especially with no route back to focus what you had focused before.

Reply:
I'll just add both /shrug